### ISSUE TITLE
it seems that the "mimes" property cannot support "image/*"

### ISSUE THREAD

**Issue by Wave-SYJ** [overtrue/vue-avatar-cropper#373]
I was using your component and tried to write this:
```html
<avatar-cropper mimes="image/*" :upload-handler="handleAvatarSelected" trigger="#pick-avatar" />
```
Then after I choosed a image file I got this error:
> File type not correct.

So I read the source code and found this in vue-avatar-cropper.vue:
```javascript
let correctType = this.mimes.split(', ').find(m => m === fileInput.files[0].type);
if (!correctType) {
  this.$emit('error', 'File type not correct.', 'user');
    return;
}
```

I think that's why I got this error. The file I choosed had a type "image/png", but "image/*" didn't have this substring.

Could you fix it? I think I have to use "image/*" because only then can I make sure the browser pops up a dialog to allow the user to choose either the photo album or the camera, not just the photo album.

**Comment by ferretwithaberet** [comment]
I fixed the `images/*` MIME not working part, but it still might not bring the user's camera, for the camera we might need to add the `capture` attribute on the file input.

**Comment by ferretwithaberet** [comment]
Closed by f9f46978d9d9a24b7f0c0190f10e218fe17f1d70.
